Sources/Paywalls/Components/PaywallPackageComponent.swift

```diff
@@ -22,30 +22,35 @@ public extension PaywallComponent {
         let type: ComponentType
         public let packageID: String
         public let isSelectedByDefault: Bool
+        public let applePromoOfferProductCode: String?
```

Comment:
There's discussion about the name of this property. I _think_ it doesn't need to be public, so we can make it `@_spi(Internal)` to only access it from RevenueCatUI. That will allow us to change the name in the future if we want without technically breaking the API.
```suggestion
        @_spi(Internal) public let applePromoOfferProductCode: String?
```